.github/workflows/check-changelog.yml

```diff
@@ -0,0 +1,33 @@
+# Checks if a file has been committed under the .changes/unreleased directory
+#
+# Skip PRs labeled with 'dependencies'
+
+on:
+  pull_request:
```

Comment:
I think I would restrict this to PRs to `main` and `pre-release`. If I create a PR onto another PR, I wouldn't want the second one to need a changie entry 🤔